servicex/servicex_utils.py

```diff
@@ -21,12 +21,15 @@ def _wrap_in_memory_sx_cache(fn):
     '''
     @functools.wraps(fn)
     async def cached_version_of_fn(*args, **kwargs):
-        assert len(args) == 2
+        assert len(args) == 2 or 3
         sx = args[0]
         from .servicex import ServiceXDataset
         assert isinstance(sx, ServiceXDataset)
         selection_query = args[1]
         assert isinstance(selection_query, str)
+        if len(args) == 3:
+            title = args[2]
+            assert isinstance(title, str)
```

Comment:
I do not think there is any need for these three lines, is there? The only key change is to line 24... And then a test that makes sure that a cached item with a title is fetched.

Reply:
Hi @gordonwatts - I realized that I've been totally forgotten this. 

Those three lines (L30-32) are just to confirm the type of argument for title is string. If it's redundant I can remove those lines :)

It seems like you already wrote tests for title - [L901 of `test_servicex.py`](https://github.com/ssl-hep/ServiceX_frontend/blob/30790674decaf140365fecb9cd9e6dce9ea49770/tests/test_servicex.py#L902). Do we need additional tests?

Reply:
Do you know how can we synchronize a title between frontend progress bar and dashboard entry?